test/smoke/process-usage/src/test/fixtures.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+const timewindow = require('abacus-timewindow');
+
+const { times } = require('underscore');
+
+const { fixturesCfg } = require('./env-config');
+
+// The scaling factor of each time window for creating the date string
+// [Second, Minute, Hour, Day, Month]
+const _slack = () =>
+  /^[0-9]+[MDhms]$/.test(fixturesCfg.slack)
+    ? {
+      scale: fixturesCfg.slack.charAt(fixturesCfg.slack.length - 1),
+      width: fixturesCfg.slack.match(/[0-9]+/)[0]
+    }
+    : {
+      scale: timewindow.dimension.min,
+      width: 10
+    };  
+
+const _initWindows = (win, dimension) => {
```

Comment:
I personally don't like the name `initWindows`. A better name could be `createInitializedWindows`. Normally I would expect `init` functions to be methods of classes and to modify the state of the object or to take the argument that needs initialization, but not to create a pre-initialized object.